src/mantine-core/src/Text/Text.styles.ts

```diff
@@ -75,7 +76,16 @@ function getLineClamp(lineClamp: number): CSSObject {
   return null;
 }
 
-function getTruncate(truncate: boolean): CSSObject {
+function getTruncate(truncate: boolean, truncateStart: boolean): CSSObject {
+  if (truncateStart) {
+    return {
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      whiteSpace: 'nowrap',
+      direction: 'rtl',
```

Comment:
This will not work for `rtl` direction. Instead of setting `direction: 'rtl'` use `direction: theme.direction === 'ltr' ? 'rtl' : 'ltr'`. In addition, please check if that works in storybook.

Reply:
@rtivital How should I make storybook use `dir: 'rtl'`?
I tried wrapping the component in `<MantineProvider theme={{dir: 'rtl'}}>` but the text is still 'ltr'.